generator/config/expression/regexFindAll.yaml

```diff
@@ -26,3 +26,74 @@ arguments:
         type:
             - string
         optional: true
+tests:
+    -
+        name: '$regexFindAll and Its Options'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/regexFindAll/#-regexfindall-and-its-options'
+        pipeline:
+            -
+                $addFields:
+                    returnObject:
+                        $regexFindAll:
+                            input: '$description'
+                            regex: !regex 'line'
+    -
+        name: 'i Option'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/regexFindAll/#i-option'
+        pipeline:
+            -
+                $addFields:
+                    returnObject:
+                        # Specify i as part of the regex field
+                        $regexFindAll:
+                            input: '$description'
+                            regex: !regex ['line', 'i']
+            -
+                $addFields:
+                    returnObject:
+                        # Specify i in the options field
+                        $regexFindAll:
+                            input: '$description'
+                            regex: 'line'
+                            options: 'i'
+            -
+                $addFields:
+                    returnObject:
+                        # Specify i in the options field
```

Comment:
This comment can be distinguished from the previous in that you're specifying the option alongside an actual regex BSON type. Likewise, maybe the first example should specify "part of a BSON regex type" instead of "part of a regex field".

Also applies to other files below.